### ISSUE TITLE
Zsh 5.7 interferes with newline 

### ISSUE THREAD

**Issue by quintrino** [bhilburn/powerlevel9k#1146]
#### Describe Your Issue
Since updating from Zsh 5.6.2 to 5.7 (The latest available with homebrew on Mac) the newline element appears to push the > end onto the following line. 

![image](https://user-images.githubusercontent.com/7043587/52029263-9e0c2a00-2566-11e9-81c5-151b68dec26c.png)
Here is my console when I revert back to zsh 5.6.2
![image](https://user-images.githubusercontent.com/7043587/52029310-c431ca00-2566-11e9-8307-b14d52e75328.png)
And here it is again after switching brew back to 5.7. 

The white tip of the custom element showing ~/.dotfiles is appearing on the next line. 

I also tested if it was just custom elements by setting the last element on the first line of my prompt as the os_icon

![image](https://user-images.githubusercontent.com/7043587/52029420-37d3d700-2567-11e9-86b0-b2740d66726f.png)

#### Have you tried to debug or fix it?

Only by reverting to 5.6.2

#### Environment Information

  - Works on 5.6.2, breaks on 5.7
  - My framework is zplugin which I used to install P9k
  - My P9k plugin version is 2019-01-23 6 ccdfdf Merge pull request #981 from pgierz/os_release_fix
  - My terminal emulator is iTerm version 3.27

#### Issues with Fonts & Icons

![image](https://user-images.githubusercontent.com/7043587/52029883-5e930d00-2569-11e9-8aec-237f985b42e5.png)

Seems to happen multiple times if I include multiple newline elements.

You may delete this section if your issue is not font / icon related.

  - Which font do you use?

SauceCodePowerline-Regular 11

  - Which [font configuration mode](https://github.com/bhilburn/powerlevel9k/wiki/About-Fonts) are you using? You can check this with (`echo $POWERLEVEL9K_MODE`).

echo $POWERLEVEL9K_MODE returns nothing but a plank line. 

  - Please share the contents of `$P9k/debug/font-issues.zsh`.

```
===== Font debugging =====
You are using iTerm2 with Font SauceCodePowerline-Regular 11

Your font settings seem to be all right. If you still have issues,
it is more likely to be a font issue than a Powerlevel9k related one.
```

  - If this is an icon problem, does the output of `$ get_icon_names` look correct?

POWERLEVEL9K_MULTILINE_FIRST_PROMPT_PREFIX: ╭─
POWERLEVEL9K_MULTILINE_LAST_PROMPT_PREFIX: ╰─
POWERLEVEL9K_MULTILINE_NEWLINE_PROMPT_PREFIX: ├─
POWERLEVEL9K_LEFT_SEGMENT_SEPARATOR: 
POWERLEVEL9K_LEFT_SUBSEGMENT_SEPARATOR: 

using get_icon_names to view the above appear to be consistent across both 5.7 and 5.6.2

**Comment by nicolasmeric** [comment]
I have the same issue on debian sid.

**Comment by panosru** [comment]
same here..

**Comment by dritter** [comment]
Thanks for the report @quintrino . I'll try to Backport the fix from the `next` branch asap.

**Comment by panosru** [comment]
@dritter sorry for late reply, it works like a charm! Thanks for your effort!

**Comment by dritter** [comment]
Thanks everyone for testing. Fixed via #1147
